pkg/handler-launcher-com/cmd/v1/cmd.pb.go

```diff
@@ -911,50 +919,52 @@ var _Cmd_serviceDesc = grpc.ServiceDesc{
-	0xd0, 0x09, 0xac, 0xa4, 0x13, 0xb1, 0x64, 0x25, 0x71, 0x7e, 0xff, 0x5e, 0x63, 0xe8, 0xb5, 0x1b,
-	0x71, 0x5a, 0xcf, 0xdd, 0x84, 0x43, 0x80, 0x4e, 0xeb, 0x58, 0xa7, 0x5f, 0x03, 0x2a, 0x24, 0xda,
-	0x82, 0x4a, 0xe8, 0xb0, 0x84, 0xb4, 0x92, 0x21, 0x45, 0x95, 0x51, 0x01, 0x7a, 0x05, 0xb3, 0x6e,
-	0xac, 0x46, 0xbd, 0xf9, 0xfc, 0xfe, 0x56, 0xb6, 0x36, 0x4f, 0xbb, 0xde, 0xdf, 0x86, 0x2f, 0x60,
-	0xb1, 0xc5, 0x2c, 0x9f, 0x44, 0xd1, 0xbf, 0x76, 0xd7, 0xd2, 0xdd, 0x6b, 0xb7, 0xd4, 0x05, 0xa8,
-	0xbd, 0x76, 0x3c, 0xd9, 0x4b, 0x88, 0xf8, 0x25, 0x54, 0x75, 0x2a, 0x3c, 0x97, 0x0b, 0x1a, 0xed,
-	0x12, 0x81, 0x61, 0x50, 0x11, 0x4f, 0xaa, 0xaa, 0xf7, 0xc3, 0x28, 0xe3, 0x50, 0x21, 0x88, 0x45,
-	0xfb, 0xe7, 0x98, 0x84, 0xf8, 0x0b, 0x2c, 0x1c, 0xb9, 0x0e, 0x61, 0x7c, 0x40, 0x79, 0x0e, 0x55,
-	0x3f, 0x79, 0x4e, 0x84, 0xde, 0xcf, 0x08, 0xed, 0x17, 0xeb, 0x83, 0xd2, 0xe8, 0x90, 0x4d, 0x05,
-	0x4a, 0x3a, 0x24, 0x11, 0xe6, 0xb0, 0x1c, 0x37, 0x68, 0x4b, 0x22, 0x45, 0xd1, 0x2e, 0x9b, 0x30,
-	0x6f, 0xde, 0xd2, 0x92, 0x56, 0x77, 0x97, 0xf0, 0x77, 0x58, 0x6a, 0x46, 0x93, 0x39, 0xe6, 0xd7,
-	0x6e, 0xd1, 0x6e, 0x4f, 0x60, 0xc9, 0x1a, 0x66, 0x25, 0x3d, 0xb3, 0x09, 0xfc, 0xa3, 0x0c, 0x75,
-	0xd5, 0xfa, 0x52, 0x50, 0xff, 0x1d, 0x13, 0xb2, 0x68, 0xfb, 0x67, 0x50, 0xb7, 0xf2, 0x78, 0x89,
-	0x84, 0xfc, 0x24, 0xfe, 0x55, 0x06, 0x4d, 0xc9, 0x78, 0xc3, 0x6c, 0x2a, 0x7a, 0x42, 0x52, 0xa7,
-	0xf0, 0xd8, 0x5f, 0x80, 0x66, 0x8d, 0x40, 0x26, 0x62, 0x46, 0xe6, 0xf7, 0x7f, 0x03, 0x54, 0x0e,
-	0x1d, 0x13, 0x9d, 0x02, 0x6a, 0xf7, 0xb8, 0x91, 0xbe, 0x35, 0xe8, 0x41, 0xee, 0x25, 0x88, 0x3f,
-	0xee, 0xb5, 0xd1, 0xda, 0x70, 0x09, 0x9d, 0xc1, 0xf2, 0x39, 0x09, 0x04, 0x9d, 0x18, 0xf0, 0x3d,
-	0xd4, 0x2f, 0xb9, 0x37, 0x51, 0xa4, 0x0e, 0xab, 0xed, 0x6e, 0x20, 0x4d, 0xf7, 0x1b, 0x9f, 0x18,
-	0xf3, 0x14, 0xd0, 0x09, 0xb3, 0xed, 0x89, 0xf1, 0xce, 0x61, 0xe5, 0x88, 0xda, 0x54, 0x4e, 0xee,
-	0xad, 0x3f, 0x40, 0x3d, 0x76, 0xbe, 0x61, 0xe4, 0xc3, 0xcc, 0xae, 0x61, 0x87, 0x1c, 0x7b, 0xe4,
-	0xd1, 0x27, 0x34, 0xd8, 0x74, 0x41, 0x7c, 0x8b, 0xca, 0x02, 0x4a, 0x3f, 0xc2, 0xfa, 0x21, 0xe1,
-	0x06, 0x1d, 0x9a, 0xe6, 0xa0, 0x41, 0x01, 0x74, 0x07, 0xd6, 0xda, 0x54, 0xa6, 0xb9, 0xea, 0x5a,
-	0x5e, 0x30, 0xa7, 0xc8, 0x70, 0x5b, 0x30, 0xd7, 0xa4, 0x32, 0xb6, 0x54, 0xb4, 0x9e, 0xa9, 0xbc,
-	0xfb, 0xcf, 0x61, 0x6d, 0x23, 0x93, 0x4e, 0x7b, 0xbd, 0x3a, 0xab, 0x85, 0x01, 0x4e, 0x19, 0xe8,
-	0x38, 0xe6, 0xa3, 0x11, 0xcc, 0x94, 0xbd, 0xe3, 0x12, 0x6a, 0x43, 0xad, 0x49, 0xe5, 0xc0, 0x8a,
-	0xc7, 0x61, 0x71, 0x26, 0x9d, 0x71, 0x71, 0x05, 0xad, 0x36, 0xa9, 0xb2, 0xbc, 0xb1, 0x3a, 0xb7,
-	0xf2, 0x81, 0x19, 0xbb, 0x2c, 0xa1, 0xcf, 0x6a, 0x04, 0x77, 0xac, 0x6b, 0x1c, 0x7a, 0x3b, 0x1f,
-	0x9d, 0x63, 0x7e, 0xb8, 0x84, 0x0e, 0x60, 0xfa, 0x9c, 0x71, 0x6b, 0x1c, 0xf3, 0x6f, 0x67, 0x7e,
-	0x30, 0xfd, 0x69, 0x2a, 0xdc, 0xbb, 0x9a, 0x51, 0x3f, 0xfa, 0x9e, 0xfe, 0x09, 0x00, 0x00, 0xff,
-	0xff, 0x67, 0xbf, 0xe6, 0x8c, 0x21, 0x0a, 0x00, 0x00,
+	// 738 bytes of a gzipped FileDescriptorProto
```

Comment:
Could you please explain what this blob is and how it is generated?
Sorry I know you didn't introduce it, but the comment is really non-descriptive...

Reply:
This is auto-generated from cmd.proto.

Reply:
This is auto-generated from cmd.proto. Nothing we can do about.